tests/baselines/reference/skipJSDocParsing/deprecated-ParseNone-file.ts.diff

```diff
@@ -38,7 +38,6 @@
 -                                "pos": 6,
```

Comment:
Does this change break JSDoc? Why are these missing?

Reply:
See the other comment, but this is misleading as this is just a removed line in a diff. These tests compare the ASTs with and without JSDoc parsing, we just so happen to not set this prop anymore (like in the other baselines).